deploy/crds/crd-certificaterequests.yaml

```diff
@@ -20,6 +20,9 @@ spec:
       - cr
       - crs
     singular: certificaterequest
+    categories:
+      - all
```

Comment:
Is there a risk that we're going to break things for people depending on `kubectl get all` returning only Kubernetes core types?
Do any other CRD projects do this?

Reply:
I agree, I think it makes sense to leave out all.

It would be great to also include `cm`, as a shorthand for `cert-manager`

Reply:
I've often wished `kubectl get all` actually returned everything rather than just core types, but it seems to be a contentious issue nonetheless https://github.com/kubernetes/kubectl/issues/151#issuecomment-594978672